docs/README.md

```diff
@@ -1,114 +1,232 @@
-| `skip_item_load`       | Whether or not to skip loading item table                                                                                                                             | false   |
+| `skip_item_load`       | Whether or not to skip loading item table.                                                                                                                            | false   |
 | `use_table_index`      | Whether or not to use a generic table-based secondary index instead of ScalarDB's secondary index.                                                                    | false   |
 | `np_only`              | Run benchmark with only new-order and payment transactions (50% each).                                                                                                | false   |
-| `rate_new_order`       | Percentage of new-order transaction. Specify all the rate parameters to use your own mix.                                                                             | N/A     |
-| `rate_payment`         | Percentage of payment transaction. Specify all the rate parameters to use your own mix.                                                                               | N/A     |
-| `rate_order_status`    | Percentage of order-status transaction. Specify all the rate parameters to use your own mix.                                                                          | N/A     |
-| `rate_delivery`        | Percentage of delivery transaction. Specify all the rate parameters to use your own mix.                                                                              | N/A     |
-| `rate_stock_level`     | Percentage of stock-level transaction. Specify all the rate parameters to use your own mix.                                                                           | N/A     |
+| `rate_new_order`       | Percentage of new-order transactions. Specify all the rate parameters to use your own mix.                                                                            | N/A     |
+| `rate_payment`         | Percentage of payment transactions. Specify all the rate parameters to use your own mix.                                                                              | N/A     |
+| `rate_order_status`    | Percentage of order-status transactions. Specify all the rate parameters to use your own mix.                                                                         | N/A     |
+| `rate_delivery`        | Percentage of delivery transactions. Specify all the rate parameters to use your own mix.                                                                             | N/A     |
+| `rate_stock_level`     | Percentage of stock-level transactions. Specify all the rate parameters to use your own mix.                                                                          | N/A     |
```

Comment:
Does `Specify all the rate parameters to use your own mix` mean the following?:

```suggestion
Specify this percentage based on your needs. The total of all rate parameters must be 100 percent.
```

Or if the total can be less than 100 percent:

```suggestion
Specify this percentage based on your needs. The total of all rate parameters must be 100 percent or less.
```